src/slider.test.aria.ts

```diff
@@ -0,0 +1,152 @@
+import { expect, test } from '@playwright/test';
+import type Slider from './slider.js';
+
+test('hide-label', async ({ page }) => {
+  await page.goto('?id=slider--slider');
+
+  await page.locator('glide-core-slider').evaluate<void, Slider>((element) => {
+    element.hideLabel = true;
+  });
+
+  await expect(page.locator('glide-core-slider')).toMatchAriaSnapshot(`
+    - text: Label
+    - group:
+      - slider "Label"
+    - spinbutton "Label"
+  `);
+});
+
+test('multiple', async ({ page }) => {
```

Comment:
Should this be somewhere in the `describe` block below?

Reply:
A bit of another tricky aria naming situation.

I want to test when `multiple` is set, but nothing else.  The existing describe blocks below test with `multiple` and `disabled` or `readonly`.  So the naming is a bit... odd.

Maybe:

```js
test.describe('multiple=${true}', () => {
  test('disabled=${false}', async ({ page }) => {
```

Or even more explicit:

```js
test.describe('multiple=${true}', () => {
  test('disabled=${false}, readonly=${false}', async ({ page }) => {
```

What do you prefer/think?